Models/Employees.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace BangazonAPI.Models
+{
+    public class Employees
+    {
+        [Key]
+        public int Id { get; set; }
+
+        [Required]
+        public Departments DepartmentsId { get; set; }
```

Comment:
Once we get the Department ID we can then get the Dept itself -- like the Customer Customer on the Order.cs class